tests/gramex.yaml

```diff
@@ -1094,23 +1094,21 @@ url:
     handler: TwitterRESTHandler
     kwargs:
       # Gramener > Gramex guide app: https://apps.twitter.com/app/12449610/keys
-      # TODO: Try regenerating these
-      key: XkCVNZD5sfWECxHGAGnlHGQFa
-      secret: yU00bx5dHYMbge9IyO5H1KeC5uFnWndntG7u6CH6O4HDZHQg0p
-      access_key: 445767176-pqio8PuGlpzULSstlZQbLhRqFJfNBjWFH5LqNRgI
-      access_secret: zVosr4wIS3aM80hErmWUkMePZ6vX830yI7VKxar6xZqgp
+      key: $TWITTER_KEY
```

Comment:
Add a commend mentioning that this is from .travis-ci.yml, and the values can be obtained from https://code.gramener.com/cto/secrets -- a new repo to be created